src/main/java/com/slobodianyk/homework4/models/User.java

```diff
@@ -0,0 +1,71 @@
+package com.slobodianyk.homework4.models;
+
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import jakarta.persistence.Entity;
+import jakarta.persistence.Table;
+import jakarta.persistence.Id;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.ManyToMany;
+import jakarta.persistence.JoinTable;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.JoinColumn;
+
+import java.util.HashSet;
+import java.util.Set;
+
+@Data
+@NoArgsConstructor
+@Entity
+@Table(name = "users")
+public class User {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    private String name;
+
+    private String email;
+
+    @ManyToMany(fetch = FetchType.EAGER)
```

Comment:
`(fetch = FetchType.EAGER)` is a bad practice, consider using `join fetch` on the repository level